manifests/f/Flow-Launcher/Flow-Launcher/1.14.1/Flow-Launcher.Flow-Launcher.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created with WinGet Releaser v2 using Komac v1.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Flow-Launcher.Flow-Launcher
+PackageVersion: 1.14.1
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: user
+InstallModes:
+- silent
+InstallerSwitches:
+  Silent: --silent
+  SilentWithProgress: --silent
+UpgradeBehavior: install
+ReleaseDate: 2023-04-20
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/Flow-Launcher/Flow.Launcher/releases/download/v1.14.1/Flow-Launcher-Setup.exe
+  InstallerSha256: A1347BF9899744A1A5B13A50AF1E86FC6891E7CC68D4EC710294025B7977DDB8

```

Hi @jjw24, this seems to be the correct hash for the x64 installer.
```suggestion
  InstallerSha256: C751AB8494E694102BB4EED086011239DEAE3A758CE99003E03AEAAD7B63EC11
```

---

Hi @rodrigoborgesdeoliveira, thanks for this. This issue is likely from our release pipeline, I will sort it out.